hypervisor/src/hypervisor/config.rs

```diff
@@ -31,6 +80,7 @@ pub struct Partition {
     pub duration: Duration,
     #[serde(with = "humantime_serde")]
     pub offset: Duration,
+    /// Repetition interval of the slice inside the MAF.
```

Comment:
also a the MAF must be cleanly dividable by this `period`

Reply:
Good point!